code/modules/cargo/expressconsole.dm

```diff
@@ -108,25 +158,28 @@
 				rank = "Silicon"
 			var/reason = ""
 			var/list/empty_turfs
-			var/area/landingzone
 			var/datum/supply_order/SO = new(pack, name, rank, ckey, reason)
 			if(!(obj_flags & EMAGGED))
 				if(SO.pack.cost <= SSshuttle.points)
 					landingzone = locate(/area/quartermaster/storage) in GLOB.sortedAreas

```

Comment:
Runtimes if the map doesn't have this area on it, preventing any use of the console.

Fall back to the area in which the console is located instead?

Reply:
If the intended area doesn't exist then falling back to the current area and logging a warning is probably sane. Look around for other code which warns.